thunder/executors/torchex.py

```diff
@@ -1721,28 +1721,7 @@ def max_pool3d_bwd_wrapper(
 ex.register_implementation(
     ltorch.max_pool3d, max_pool3d, checker=_always_executable, grad_transform=max_pool3d_bwd_wrapper
 )
-
-
-def adaptive_avg_pool2d_bwd_wrapper(
```

Comment:
@kiya00, can you please remind me why this change was required?

Reply:
The reason is [here](https://github.com/Lightning-AI/lightning-thunder/pull/837#issuecomment-2263378611), we could solve the problem described in that comment, or we could register it in another way like this and avoid hitting the error